nano/node/nodeconfig.hpp

```diff
@@ -110,6 +110,7 @@ class node_flags final
 {
 public:
 	std::vector<std::string> config_overrides;
+	bool disable_active_transactions{ false };
```

Comment:
Isn't this more like disable[_active_transaction]_request_loop? Not disabling elections completely

Reply:
Good point, changed.